RELEASE_NOTES.md

```diff
@@ -1,6 +1,6 @@
 # Release notes
 
-## New in git-machete 3.9.2
+## New in git-machete 3.10.0
 
 - added: configurable rendering of `status` command
```

Comment:
Pls name the new config explicitly in the release notes (that's a general rule for any good release notes; details on what the feature does aren't necessary, but at least it should be unambiguous WHAT has been added/changed)